srcpkgs/sigi/template

```diff
@@ -1,14 +1,14 @@
 # Template file for 'sigi'
 pkgname=sigi
-version=3.4.3
+version=3.5.0
 revision=1
 build_style=cargo
 short_desc="Organization tool for people who hate organization"
 maintainer="J.R. Hill <hiljusti@so.dang.cool>"
 license="GPL-2.0-only"
 homepage="https://github.com/hiljusti/sigi"
```

Comment:
```suggestion
homepage="https://github.com/sigi-cli/sigi"
```
If you want, you could write the distfiles as `distfiles="https://static.crates.io/crates/sigi/sigi-${version}.crate"`

Reply:
Oh thanks! I missed this

Reply:
and I didn't know about the static crates link, I'll try that out too